tls/test_tls_cert.py

```diff
@@ -0,0 +1,405 @@
+"""
+Tests for basic x509 handling: certificate loading and getting a valid request
+and response, stale certificates and certificates with unsupported algorithms.
+ECDSA-SHA256-SECP256R1 is the default certificate, so do not test it.
```

Comment:
It's defaults in Tempesta, that can be unlilely changed in the future. I would like to have explicit test for it.